shaders/Membrane.gdshader

```diff
@@ -30,6 +30,10 @@ void vertex(){
         * wigglyNess * size;
     VERTEX.z += sin(worldVertex.x * movementWigglyNess - TIME / 4.0f) / 10.f
         * wigglyNess * size;
+
+    float cellOffset = (NODE_POSITION_WORLD.x + NODE_POSITION_WORLD.z);
+    VERTEX.y += (sin(cellOffset * movementWigglyNess + VERTEX.z + TIME / 4.0f) + 1.0f)
```

Comment:
I assume the y change here is not affecting the EngulfEffect shader as this doesn't change like the 2D shape at the height the wave effect is at.